cypress/integration/spec.js

```diff
@@ -54,7 +64,9 @@ describe('Chart Explorer', function() {
     ).click()
 
     cy.get('#react-select-2-option-3').click()
+  })
 
+  it('Should perform basic functionalities', function() {
```

Comment:
i think it would be awesome to have a little more detail in this test name. like, which actions specifically does "basic" imply?